apps/console/src/features/roles/components/wizard/role-user-assign.tsx

```diff
@@ -579,30 +579,77 @@ export const AddRoleUsers: FunctionComponent<AddRoleUserProps> = (props: AddRole
-                                                                                floated="left"
-                                                                                image={ user.profileUrl }
-                                                                            />
-                                                                        </Table.Cell>
-                                                                        <Table.Cell>
-                                                                            { user.userName }
-                                                                        </Table.Cell>
-                                                                    </Table.Row>
-                                                                );
+                                                            assignedUsers?.map((user: RolesMemberInterface) => {
+                                                                if (user.orgId) {
+                                                                    return (
+                                                                        <Table.Row key={ user.value }>
+                                                                            <Table.Cell collapsing>
+                                                                                <UserAvatar
+                                                                                    data-testid={ `${ testId }-users
+                                                                                    -list-${ user.display }-avatar` }
+                                                                                    name={ 
+                                                                                        user.display
+                                                                                    }
+                                                                                    size="mini"
+                                                                                    floated="left"
+                                                                                />
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                { user.display }
+                                                                            </Table.Cell>
+                                                                            <Table.Cell>
+                                                                                {
+                                                                                    user.orgName
+                                                                                    === "Super" ? "Asgardeo" : 
```

Comment:
Let's not use hardcoded values here. Better to use a config instead.

Reply:
Resolved with https://github.com/wso2/identity-apps/pull/3887/commits/c2eba0e23012ea071441b09a2445e2fd1b241868